src/runner.js

```diff
@@ -305,10 +306,19 @@ const Runner = /** @class */ (function () {
 					const message = (err && err.message) ? err.message : '';
 
 					def.status = 'rejected';
-					client.contaminate({
-						reason: 'error-thrown',
-						details: message
-					});
+
+					if (options.debug) {
+						client.contaminate({
+							reason: 'error-thrown',
+							details: message,
+							stack: err.stack
```

Comment:
does the stack value make it to the events and analytics call? I don't believe the client is expecting that argument

Reply:
I was focusing on seeing it in CloudWatch, which you can.

Are you talking about tracing it back to Snowflake? Or something else?

Reply:
It made it through to `CONTEXT_STAGES`. 
<img width="676" alt="image" src="https://github.com/evolv-ai/asset-manager/assets/78709720/596f0499-9817-4b29-91d6-040d49654524">